Documentation/network/concepts/masquerading.rst

```diff
@@ -72,7 +72,13 @@ The eBPF-based masquerading can masquerade packets of the following IPv4 L4 prot
 
 - TCP
 - UDP
-- ICMP (only Echo request and Echo reply)
+- ICMP
+
+.. note::
+
+    For ICMP, support is limited to Echo request, Echo reply, and
+    start to Cilium v1.14.0 the error message Destination unreachable
+    fragmentation needed.
```

Comment:
I don't understand this part of the sentence, could you please rephrase?

I don't think we need to mention v1.14 in this doc. This commit won't be backported, so this note should only appear on versions v1.14+ of the documentation (and `.../latest` until 1.14 is out).

Reply:
Yes... I updated the phrasing. Thank you for the review and information. I hope this will be better.

Reply:
_(Feel free to share with me any better suggestion)_